src/kata6.joinNames.js

```diff
@@ -1,3 +1,13 @@
-const joinNames = (namesObj) => {};
+const joinNames = (namesObj) => {
+  let string = "";
+  for (let i = 0; i < namesObj.length; i++) {
+    if (i === namesObj.length - 2) {
+      string += `${namesObj[i].name} & ${namesObj[i + 1].name}`;
+      break;
+    }
+    string += `${namesObj[i].name}, `;
+  }
```

Comment:
instead of using a `break` here, you could instead change it to an `if else` and change your condition to `i < namesObj.length - 1`, e.g.
```
  for (let i = 0; i < namesObj.length - 1; i++) {
    if (i === namesObj.length - 2) {
      string += `${namesObj[i].name} & ${namesObj[i + 1].name}`;
    } else {
      string += `${namesObj[i].name}, `;
    }
  }
```